`ppv-lite86` fails to compile when zerocopys `derive` feature is active

The yanked version `0.2.19` of `ppv-lite86` fails to compile when the `derive` feature of `zerocopy` is active. The feature is not enabled by `ppv-lite86`, but can be transitively enabled by any other crate in the build tree.

Try to compile the following minimal project:

<details><summary>Minimal project</summary>

`Cargo.toml`
```toml
[package]
name = "test"
version = "0.1.0"
edition = "2021"

[dependencies]
ppv-lite86 = { version = "=0.2.19", features = ["no_simd"] }
zerocopy = { version = "0.7", features = ["derive"] }
```
--------

`src/lib.rs`
```rust
use ppv_lite86 as _;
use zerocopy as _;
```
</details>

**Stacktrace**
```
   Compiling ppv-lite86 v0.2.19
error[E0252]: the name `AsBytes` is defined multiple times
 --> /home/dev/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ppv-lite86-0.2.19/src/generic.rs:7:23
  |
6 | use zerocopy::{AsBytes, FromBytes};
  |                ------- previous import of the macro `AsBytes` here
7 | use zerocopy_derive::{AsBytes, FromBytes, FromZeroes};
  |                       ^^^^^^^--
  |                       |
  |                       `AsBytes` reimported here
  |                       help: remove unnecessary import
  |
  = note: `AsBytes` must be defined only once in the macro namespace of this module

error[E0252]: the name `FromBytes` is defined multiple times
 --> /home/dev/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ppv-lite86-0.2.19/src/generic.rs:7:32
  |
6 | use zerocopy::{AsBytes, FromBytes};
  |                         --------- previous import of the macro `FromBytes` here
7 | use zerocopy_derive::{AsBytes, FromBytes, FromZeroes};
  |                                ^^^^^^^^^--
  |                                |
  |                                `FromBytes` reimported here
  |                                help: remove unnecessary import
  |
  = note: `FromBytes` must be defined only once in the macro namespace of this module

For more information about this error, try `rustc --explain E0252`.
error: could not compile `ppv-lite86` (lib) due to 2 previous errors
```

This problem was introduced by ee6c7d4056780ee58593597b739bb501be567e03. [`probe-rs` had a similar issue](https://github.com/probe-rs/probe-rs/issues/2613).

I am willing to provide a pull request, but there is two possible solutions to the problems. Which one do you prefer?

* Remove the dependency on `zerocopy_derive` and enable `zerocopy`s  `derive` feature. This is the solution that was adopted by [`probe-rs`](https://github.com/probe-rs/probe-rs/pull/2614).
* Use the traits under a different name:
  ```diff
  - use zerocopy::{AsBytes, FromBytes};
  + use zerocopy::{AsBytes as _, FromBytes as _}; 
  ```

--- a/hashes/groestl/Cargo.toml
+++ b/hashes/groestl/Cargo.toml
@@ -15,8 +15,7 @@ rust-version = "1.61"
 block-buffer = "0.9"
 digest = "0.9"
 lazy_static = { version = "1.2", optional = true }
-zerocopy = { version = "0.7", features = ["simd"] }
-zerocopy-derive = "0.7"
+zerocopy = { version = "0.7", features = ["simd", "derive"] }
 
 [dev-dependencies]
 digest = { version = "0.9", features = ["dev"] }

--- a/hashes/groestl/src/compressor.rs
+++ b/hashes/groestl/src/compressor.rs
@@ -2,7 +2,7 @@ use block_buffer::generic_array::typenum::{U128, U64};
 use block_buffer::generic_array::GenericArray;
 use core::arch::x86_64::*;
 use core::ops::BitXor;
-use zerocopy_derive::{AsBytes, FromBytes, FromZeroes};
+use zerocopy::{AsBytes, FromBytes, FromZeroes};
 
 trait Map2 {
     type Output;

--- a/utils-simd/ppv-lite86/Cargo.toml
+++ b/utils-simd/ppv-lite86/Cargo.toml
@@ -11,8 +11,7 @@ categories = ["cryptography", "no-std"]
 rust-version = "1.61"
 
 [dependencies]
-zerocopy = { version = "0.7", features = ["simd"] }
-zerocopy-derive = "0.7"
+zerocopy = { version = "0.7", features = ["simd", "derive"] }
 
 [badges]
 travis-ci = { repository = "cryptocorrosion/cryptocorrosion" }

--- a/utils-simd/ppv-lite86/src/generic.rs
+++ b/utils-simd/ppv-lite86/src/generic.rs
@@ -3,8 +3,7 @@
 use crate::soft::{x2, x4};
 use crate::types::*;
 use core::ops::*;
-use zerocopy::{AsBytes, FromBytes};
-use zerocopy_derive::{AsBytes, FromBytes, FromZeroes};
+use zerocopy::{AsBytes, FromBytes, FromZeroes};
 
 #[repr(C)]
 #[derive(Clone, Copy, FromBytes, AsBytes, FromZeroes)]

--- a/utils-simd/ppv-lite86/src/soft.rs
+++ b/utils-simd/ppv-lite86/src/soft.rs
@@ -4,7 +4,7 @@ use crate::types::*;
 use crate::{vec128_storage, vec256_storage, vec512_storage};
 use core::marker::PhantomData;
 use core::ops::*;
-use zerocopy_derive::{AsBytes, FromBytes, FromZeroes};
+use zerocopy::{AsBytes, FromBytes, FromZeroes};
 
 #[derive(Copy, Clone, Default, FromBytes, AsBytes, FromZeroes)]
 #[repr(transparent)]

--- a/utils-simd/ppv-lite86/src/x86_64/mod.rs
+++ b/utils-simd/ppv-lite86/src/x86_64/mod.rs
@@ -2,7 +2,7 @@
 
 use crate::types::*;
 use core::arch::x86_64::{__m128i, __m256i};
-use zerocopy_derive::{AsBytes, FromBytes, FromZeroes};
+use zerocopy::{AsBytes, FromBytes, FromZeroes};
 
 mod sse2;
 

--- a/utils-simd/ppv-lite86/src/x86_64/sse2.rs
+++ b/utils-simd/ppv-lite86/src/x86_64/sse2.rs
@@ -9,8 +9,7 @@ use core::marker::PhantomData;
 use core::ops::{
     Add, AddAssign, BitAnd, BitAndAssign, BitOr, BitOrAssign, BitXor, BitXorAssign, Not,
 };
-use zerocopy::transmute;
-use zerocopy_derive::{AsBytes, FromBytes, FromZeroes};
+use zerocopy::{transmute, AsBytes, FromBytes, FromZeroes};
 
 macro_rules! impl_binop {
     ($vec:ident, $trait:ident, $fn:ident, $impl_fn:ident) => {
@@ -1385,8 +1384,7 @@ pub mod avx2 {
     use core::arch::x86_64::*;
     use core::marker::PhantomData;
     use core::ops::*;
-    use zerocopy::transmute;
-    use zerocopy_derive::{AsBytes, FromBytes, FromZeroes};
+    use zerocopy::{transmute, AsBytes, FromBytes, FromZeroes};
 
     #[derive(Copy, Clone, FromBytes, AsBytes, FromZeroes)]
     #[repr(transparent)]

